src/LiveStream.js

```diff
@@ -92,9 +94,22 @@ class EluvioLiveStream {
         objectId: conf.objectId
       });
 
+      if (mainMeta.live_recording_config == undefined || mainMeta.live_recording_config.url == undefined) {
+        status.state = "unconfigured";
+        return status;
```

Comment:
This may be an invalid object - probably best to call it 'invalid' or throw an error